docs/configuration/network_filters/redis_proxy_filter.rst

```diff
@@ -0,0 +1,91 @@
+.. _config_network_filters_redis_proxy:
+
+Redis proxy
+===========
+
+Redis :ref:`architecture overview <arch_overview_redis>`.
+
+.. code-block:: json
+
+  {
+    "name": "redis_proxy",
+    "type": "read",
+    "config": {
+      "cluster_name": "...",
+      "conn_pool": "{...}",
+      "stat_prefix": "..."
+    }
+  }
+
+cluster_name
+  *(required, string)* Name of cluster from cluster manager.
+  See the :ref:`configuration section <arch_overview_redis_configuration>` of the architecture overview for recommendations
+  on configuring the backing cluster.
+
+conn_pool
+  *(required, object)* Connection pool configuration.
+
+stat_prefix
+  *(required, string)* The prefix to use when emitting :ref:`statistics <config_network_filters_redis_proxy_stats>`.
+
+Connection pool configuration
+-----------------------------
+
+.. code-block:: json
+
+  {
+    "op_timeout_ms": "...",
+  }
+
+op_timeout_ms
+  *(required, integer)* Per-operation timeout in milliseconds.
```

Comment:
Can you add a bit of text on how the timeout actually works. If you describe this in the arch overview feel free to link to that.